about.html

```diff
@@ -0,0 +1,142 @@
+            </nav>
+            <div class="menu-bar d-block d-md-none container-fluid"><i class="fa-solid fa-bars p-3" id="open-btn"></i></div>
+            <nav id="nav-wrapper" class="d-md-none">
+                <div class="header-logo-container">
+                    <a href="./index.html"><img src="./images/tech-conference-logo.webp" class="header-logo d-none" alt=""></a>
+                    <i class="fa-solid fa-xmark fs-1 ps-5 pt-4" id="cross-btn"></i>
+                </div>
+                <ul class="nav-items">
+                    <li><a href="./index.html">Home</a></li>
+                    <li><a href="./about.html">About</a></li>
+                    <li><a href="#intro-setcion">Program</a></li>
+                    <li><a href="./index.html#main-program-section">Join</a></li>
+                    <li><a href="./index.html#sponsor-section">Sponsors</a></li>
+                    <li><a href="#past-summit">News</a></li>
+                    <li><a href="#/">CC Campaign</a></li>
+                </ul>
+            </nav>
+        </header>
+
+        <!-- INTRO SECTION START  -->
+        <section id="intro-section">
+            <div class="row">
+                <div class="col" id="about-intro-header">
+                    <h3 class="text-center d-block d-md-none">"Hello!Sharing world"</h3>
+                    <h1>Tech. Innovation<br>Conference 2023</h1>
+                    <div class="line d-none d-md-block"></div>
+                </div>
+            </div>
+            <div class="welcome-text-container">
+                <p class="text-center bg-white" id="welcome-text">The Tech Innovation Conference brings together the community of
+                    experts, academics, and activists who comprise the Creative Commons
+                    affiliate network in a different country every two.
+                    <span class="d-none d-md-block text-center">This year, we hope to expand our invitation lists including
+                        organizations and individuals who want to work with us on shared projects that advanced the cause of the Commons, free culture and open knowledge.
+                        So if you're active and engaged in the worlds of open content and knowledge - free software advocates. Wikipedians. Open Knowledge,galleries,libraries,museums,archives,governents and foundations,lawyers,and activists - we hope you'll consider joining us this year to build a stronger,
+                        more vibrant commons together.</span>
+                </p>
+            </div>
+            <div id="contact-container">
+                <p id="" class="text-center contact-text">Please contact us per Email for any further questions about CC Global Summit 2023!</p>
+                <a href="" class="text-center fw-bold d-block text-decoration-none text-black"><span id="contact-email">techconference2023@newyork.org</span></a>
+                <div class="highlight"></div>
+            </div>
+        </section>
```

Comment:
- [ ] Your background images do not match the figma design on the desktop. Kindly match them to fit the design.
![scrn](https://github.com/HtetWaiYan7191/capstone-project/assets/99656687/44be5202-b3ef-4c69-8950-faa1f8201b0b)